hm-post-repeat.php

```diff
@@ -117,7 +120,14 @@ function publish_box_ui() {
 function post_states( $post_states, $post ) {
 
 	if ( is_repeating_post( $post->ID ) ) {
-		$post_states['hm-post-repeat'] = __( 'Repeating', 'hm-post-repeat' );
+
+		// If the schedule has been removed since publishing, let the user know.
+		if ( get_repeating_schedule( $post->ID ) ) {
+			$post_states['hm-post-repeat'] = __( 'Repeating', 'hm-post-repeat' );
+		} else {
+			$post_states['hm-post-repeat'] = __( 'Invalid Repeating Schedule', 'hm-post-repeat' );
```

Comment:
@noplanman I'm not convinced we need this additional `post_state`, the post will simply stop repeating if a schedule is removed, it seems pretty edge case and maybe fine to just fail silently. If we do want to handle the error condition maybe we should instead / also show a message in the publish metabox, here: https://github.com/humanmade/repeatable-posts/pull/18/files#diff-eedfb6e9d8d52eb8169e5eef4b936121R89